src/index.js

```diff
@@ -49,7 +50,7 @@ export function pitch(request) {
 
   worker.options = {
     filename,
-    chunkFilename: `[id].${filename}`,
+    chunkFilename: this._compilation.outputOptions.chunkFilename,
```

Comment:
By default this should come from https://webpack.js.org/configuration/output/#outputchunkfilename